beeline/trace.py

```diff
@@ -49,8 +50,8 @@ def __init__(self, client):
 
         self.presend_hook = None
         self.sampler_hook = None
-        self.http_trace_parser_hook = beeline.propagation.honeycomb.http_trace_parser_hook
-        self.http_trace_propagation_hook = beeline.propagation.honeycomb.http_trace_propagation_hook
+        self.http_trace_parser_hook = beeline.propagation.default.http_trace_parser_hook
+        self.http_trace_propagation_hook = beeline.propagation.default.http_trace_propagation_hook
```

Comment:
Ooo. I didn't add an outgoing hook to Ruby's new default propagator. That's a good idea. 👍🏻